frontend/pages/_app.tsx

```diff
@@ -15,10 +16,16 @@ const App = ({ Component, pageProps }: AppProps) => {
     <QueryClientProvider client={queryClient}>
       <RecoilRoot>
         <Header />
-        <Component {...pageProps} />
+        <Box>
+          <Component {...pageProps} />
+        </Box>
       </RecoilRoot>
     </QueryClientProvider>
   );
 };
 
 export default App;
+
+const Box = styled.main`
+  padding-top: 100px;
```

Comment:
```suggestion
  padding-top: 80px;
```
Header  스타일의 height가 80px이라 맞춰주는 것이 좋을 것 같습니다.

Reply:
수정하였습니다~